__tests__/app.test.js

```diff
@@ -184,6 +184,22 @@ describe("GET: /api/topics", () => {
             expect(body.msg).toBe('Bad Request')
             })
     })
+    test("GET:200 accepts a topic query which filters the articles by the topic value specified", () => {
+        return request(app).get("/api/articles?topic=mitch")
+        .expect(200)
+        .then(({body}) => {
+            body.articles.forEach((article) => {
```

Comment:
I would add a test for the length of the array before using a forEach. This way if the array is empty then the tests inside the forEach would be skipped but the length test outside would still run